src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java

```diff
@@ -55,15 +55,15 @@ public void create() {
   @Override
   public void storeObject(final String schema, final String key, final Object value)
       throws CTPException, DataStoreContentionException {
-    log.with(schema).with(key).debug("Saving object to Firestore");
+    log.with("schema", schema).with("key", key).info("Saving object to Firestore");
 
     // Store the object
     ApiFuture<WriteResult> result = firestore.collection(schema).document(key).set(value);
 
     // Wait for Firestore to complete
     try {
       result.get();
-      log.with(schema).with(key).debug("Firestore save completed");
+      log.with("schema", schema).with("key", schema).info("Firestore save completed");
```

Comment:
I think this says schema where it should say key. I suggest this:
```suggestion
      log.with("schema", schema).with("key", key).info("Firestore save completed");
```